### DISCUSSION TITLE
Elementary os 8 become black screen after grub

### DISCUSSION THREAD

**Discussion by skytetsu** [elementary/triage#594] [Q&A]
I choose elementary and screen get black, I can boot system in recovery mode. When I do recovery mode the volumes say they not OK (bad). I install it in the latest RC. I wondered if is fixed in the dailies. My PC is a dell laptop with a Intel Iris touch screen. The system boot normal after do the recovery mode; but for boot I need to do recovery mode everytime

**Comment by doobes** [comment]
I'm having not dissimilar issues...[https://github.com/orgs/elementary/discussions/593](url)

**Comment by jeremypw** [comment]
I had this problem too on a multiboot system - I now run OS8 in a VM without that problem (there are other drawbacks).  I found running OS7.1 and OS8 multiboot causes problems. Are you multibooting or can you let OS8 setup on  a clean disk?  Hopefully this will be fixed soon ...

**Reply by skytetsu** [reply]
Yes is a dual boot with windows

**Comment by doobes** [comment]
This was a bare metal install on a Dell laptop. Something happened with a low battery dictated shutdown.

I'm not going to do any further work until someone responds.
